docs/guides/qiskit-addons-sqd.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: Sample-based quantum diagonalization
+description: Overview of the Sample-based quantum diagonalization (SQD) workflow
+---
+
+# Sample-based quantum diagonalization (SQD)
+
+Sample-based quantum diagonalization (SQD) is a technique for finding eigenvalues and eigenvectors of quantum operators, such as the Hamiltonian of a quantum system, using quantum and distributed classical computing together.
+
+Classical computing is used to process samples obtained from a quantum processor, and to project and diagonalize a target Hamiltonian in a subspace spanned by them. This allows SQD to be robust to samples corrupted by quantum noise and manage large Hamiltonians, such as chemical systems with millions of interacting terms, beyond the reach of exact diagonalization methods.
+
+The SQD tool can target Hamiltonians expressed as linear combinations of Pauli operators, or second-quantized fermionic operators. The input samples are obtained by quantum circuits defined by the user, which are believed to be good representations of eigenstates (for example the ground state) of a target operator. The convergence rate of SQD as a function of the number of samples improves with the sparseness of the target eigenstate.
+
+## Install SQD package
+
+There are two ways to install the SQD package. Via PyPI and building from source. We also recommend installing these packages in a [virtual environment](https://docs.python.org/3.10/tutorial/venv.html) to ensure separation between package dependencies.
+
+### Install from PyPI
+
+The most straightforward way to install the `qiskit-addon-sqd` package is via PyPI.
+
+```bash
+pip install qiskit-addon-sqd
+```
+
+### Build from source
```

Comment:
I'm wondering if we should put the build from source section inside a twisty? I think it's only relevant for a small subset of users